layouts/shortcodes/codepen.html

```diff
@@ -0,0 +1,9 @@
+<p data-height="{{ .Get "height" | default "573" }}"
+   data-theme-id="light"
+   data-slug-hash="{{ .Get "slug"}}"
+   data-default-tab="result"
+   data-user="{{ .Get "user" | default "Serabe" }}"
+   data-embed-version="2"
+   data-pen-title="{{ .Get "title" | default "sourced"}}"
+   class="codepen"></p>
+<script async src="https://production-assets.codepen.io/assets/embed/ei.js"></script>
```

Comment:
In case the user uses two codepens in the same blog, the script will load twice.
(using the same 3rd party script sometimes fails, could you confirm that codepen does not fail if loaded twice?)